docs/ref/utils.txt

```diff
@@ -147,6 +147,11 @@ The functions defined in this module share the following properties:
     ``P4DT1H15M20S`` which is equivalent to ``4 1:15:20``) or PostgreSQL's
     day-time interval format (e.g. ``3 days 04:05:06``).
 
+.. versionchanged:: 6.1
+
+    ISO 8601 duration parsing now supports weeks (``PnW``). Years (``Y``) and
+    months (``M``) remain unsupported.
```

Comment:
I'll move some of this into the text above. The rule of thumb is that the versionchanged directive shouldn't contain any information we want to keep forever, as when 6.1 goes EOL, we drop this whole directive.